crates/node/src/runtime/dkg_module.rs

```
@@ -537,9 +308,33 @@ impl Handler<EventMessage> for DkgModule {
                 match result {
                     Ok(status) => {
                         info!("DKG Completion status {:?}", status);
+                        if let Some(public_key_set)=self.dkg_engine.dkg_state.public_key_set.as_ref(){
+                            let _ = self
+                                .broadcast_events_tx
+                                .send(EventMessage::new(
+                                    None,
+                                    Event::QuorumKey(
+                                        public_key_set.public_key().to_bytes().to_vec()
+                                    ),
+                                ))
+                                .await;
+
+                            //Namespace Registration
+                            let _ = self
```

Dont do this, report the error via a match expression instead.

---

+ 1 on this

I'll be going through these shortly to try and add error handling where it's missing, but in the future failing silently can make release failures harder to debug!

---

I have already addressed it in this PR . But do check if i had accidentally missed some out